news/_posts/bands_image_analysis.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Quantifying Electrophoresis Gel Bands with QuPath and Galaxy"
+author: "Reyhaneh Tavakoli-Koopaei"
+contributions:
+  authorship: [reytakop, dianichj]
+tags: [GTN, imaging, electrophoresis, QuPath, Galaxy]
+layout: news
+date: "2025-08-18"
+cover: "news/images/spoc_images/bands_image_analysis.png"
+coveralt: "Electrophoresis gel bands visualized and quantified using bioimaging workflows"
+subsites: [all,global,esg,eu]
```

Comment:
You can add the link to your tutorial to the metadata like this:
`tutorial: topics/.../...`